packages/framework/src/Resources/translations/validators.cs.po

```diff
@@ -4,6 +4,9 @@ msgstr ""
 "Content-Transfer-Encoding: 8bit\n"
 "Language: cs\n"
 
+msgid ""
```

Comment:
I am wondering how to avoid this. It is because of the empty strings in the constraint messages, right? I see it is not possible to set the message to `null`, as then the default "This value is not valid" message is used. However, the messages are automatically extracted thanks to our implementation, so maybe we could tweak our `ConstraintMessageExtractor` to skip the empty strings :thinking: